python-checks/src/test/resources/checks/incompatibleOperands/arithmetic.py

```diff
@@ -59,12 +59,12 @@ def __neg__(self):
 
 def builtin_noncompliant():
   1 + "1"  # Noncompliant
-  "1" + 1  # Noncompliant
+  "1" + 1  # FN Noncompliant
```

Comment:
Here, `str.__add__` is ambiguous and has 2 possible symbols. It is not known to have incompatible type.
`int.__radd__` is not ambiguous and known to have incompatible type.